src/nanoemoji/write_tidysvg.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tidy up an svg enough resvg won't be grumpy about it.
+
+https://github.com/googlefonts/nanoemoji/issues/208
+"""
+from absl import app
+from picosvg.svg import SVG
+
+
+def main(argv):
+    src_file, dest_file = argv[1:]
+    svg = SVG.parse(src_file)
+    svg.absolute(inplace=True).remove_unpainted_shapes(inplace=True)
```

Comment:
i think it would be better to have resvg render the original SVGs are they are, without introducing any changes. If we want we can fix up the sources as needed. The purpose of the comparison is check that the output of our build pipeline renders the same as the inputs would render, if we are also modifying the input (even with a minor relatively change like this) then we are no longer comparing original vs generated.